src/main/resources/application.yaml

```diff
@@ -52,7 +52,7 @@ ftp:
   privateKey: ${FTP_PRIVATE_KEY}
   publicKey: ${FTP_PUBLIC_KEY}
   downtime:
-    from: ${FTP_DOWNTIME_FROM:16:00}
+    from: ${FTP_DOWNTIME_FROM:15:00}
     to: ${FTP_DOWNTIME_TO:17:00}
```

Comment:
missed second time?

Reply:
@nitinprabhu pointed out it doesn't really matter as long as that specific hour is skipped 👍